composer.json

```diff
@@ -31,7 +31,7 @@
 	"require": {
 		"php": ">=5.3.0",
 		"data-values/data-values": "~1.0|~0.1",
-		"data-values/interfaces": "~0.1",
+		"data-values/interfaces": "~0.1.5",
```

Comment:
This change might not be doing what you wanted to achieve. What are you trying to do here?